app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -224,7 +224,7 @@ class MainActivity : ComponentActivity() {
         resultsMap ->
         var permissionGranted = false
         resultsMap.forEach {
-            if (it.value == true) {
+            if (it.value) {
```

Comment:
Removed redundancy. == true was not necessary.

Reply:
Agreed